### PR TITLE
Ignore test_system_health for 201911 images

### PR DESCRIPTION
**Pull request by antoninamelnyk**
Signed-off-by: Antonina Melnyk antoninax.melnyk@intel.com

### Description of PR

Summary:
Skip `test_system_health` testcase on 201911 images. This testcase is primarily targeted to be executed on 202012 images.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?
`test_system_health` tests are failing on 201911 images because this feature is not supported by 201911 release.

#### How did you do it?
Added a module scoped skip fixture.

#### How did you verify/test it?
Executed the test on a 201911 image, and it was successfully skipped:
```
system_health/test_system_health.py::test_service_checker SKIPPED                                                                                                                                       [ 25%]
system_health/test_system_health.py::test_device_checker SKIPPED                                                                                                                                        [ 50%]
system_health/test_system_health.py::test_external_checker SKIPPED                                                                                                                                      [ 75%]
system_health/test_system_health.py::test_system_health_config SKIPPED                                                                                                                                  [100%]
```

### CODE REVIEW

**Comment by antoninamelnyk** [comment]
@yxieca @wangxin @shlomibitton Please review

**Comment by antoninamelnyk** [comment]
retest vsimage please

**Review by wangxin — APPROVED** [review]

File: tests/system_health/test_system_health.py [hunk 1]
```diff
@@ -49,6 +50,13 @@
 EXPECT_PSU_INVALID_VOLTAGE = '{} voltage is out of range'
 
 
+@pytest.fixture(autouse=True, scope="module")
+def check_image_version(duthost):
+    """Skip the test for unsupported images."""
+    pytest_require("201911" not in duthost.os_version, "Test not supported for 201911 images. Skipping the test")
```
Comment (wangxin) [inline tests/system_health/test_system_health.py:56]: If the 2020 branch has newer kernel version, then it is more robust to skip this test on 201911 branch by kernel_version.

Example: 
https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/cli/test_show_platform.py#L118

Comment (antoninamelnyk) [reply tests/system_health/test_system_health.py]: Great suggestion. Updated. Thanks.

**Pull request merged**